dts/bindings/gpio/semtech,sx1509b-gpio.yaml

```diff
@@ -21,6 +21,22 @@ properties:
       required: true
       const: 16
 
+    init-out-low:
```

Comment:
The title and description of this commit should be corrected. The commit does not actually change anything in a board named "thingy52", it adds new bindings for properties of "semtech,sx1509b" nodes and handling of these in the gpio_sx1509b driver.